SigmaTcpDaemon/Main.cpp

```diff
@@ -0,0 +1,62 @@
+/*Copyright 2017 Bang & Olufsen A/S
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.*/
+
+
+#include "SigmaTcpServer.h"
+#include "SpiCommunication.h"
+#include "EepromHandler.h"
+#include <iostream>
+#include <exception>
+
+int main()
+{
+ 	SigmaTcpServer tcpServer;
+	SpiCommunication spiCommIf;
+	EepromHandler eepromHandler;
```

Comment:
Cannot see that we are actually using the EepromHandler? Maybe that will be used in the future?